components/main-modal.tsx

```
@@ -0,0 +1,28 @@
+'use client'
+
+import Account from '@/components/account'
```

ESLint

[error] 'Account' is defined but never used.

(@typescript-eslint/no-unused-vars)